docs/lib/storage/fragments/native_common/configureaccess/common.md

```diff
@@ -0,0 +1,46 @@
+ When adding the Storage category, you configure the level of access authenticated and guest users have to your S3 bucket. Additionally, when uploading files using the Storage category, you can specify the access level for that file to be either guest(public), protected, or private.
+
+- **Guest** Accessible by all users
+- **Protected** Readable by all users, but only writable by the creating user
+- **Private** Readable and writable only by the creating user
+
+For protected and private access, the `[USER_ID]` below corresponds to the unique ID of the authenticated user. Check out [Authentication](~/lib/auth/getting-started.md) to get started with signing up and signing in a user. Once the user has signed in, the `[USER_ID]` can be retrieved from the session by accessing the identity id. See [Accessing credentials](~/lib/auth/access_credentials.md) to retrieve the identity id, and use this as the unique ID of the authenticated user.
+
+<amplify-callout>
+
+The default access level for the Storage category is **guest**. Unless you specify otherwise, all uploaded files will be publicly available for all users.
```

Comment:
```suggestion
The default access level for the Storage category is **guest**. Unless you specify otherwise, all uploaded files will be publicly available for all users of your application.
```

Do we think this helps to provide further clarification that this isn't "public"? I know there has been some confusion about that in the past.

Reply:
as part of this PR, i moved the sequences of pages to 
1. getting started
2. concepts 
3. upload/download/list, etc
4. file access levels

I think more information/improvements can go into concepts, and added onto your suggestion here

```
The default access level for the Storage category is **guest**. Unless you specify otherwise, all uploaded files will be publicly available for all users of your application. This means a user that is using your application and not signed in will have access. A user that is not using your application will not be able to access your files.
```